website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,193 @@
+
+“Show selected” will change the list to display only selected list items while toggling the label of the button to “Show all.”
+
+![Flow showing the behavior of the show selected button and how it toggles the list to only display selected list items](/assets/components/form/super-select/super-select-flow-select-all.png)
+
+If “Show selected” is used when no selections have been made, a message will appear to let the user know.
+
+![Flow showing the behavior of the show selected button when no selections have been made yet.](/assets/components/form/super-select/super-select-flow-no-selections-select-all.png)
+
+Additionally, “Clear selected” will wipe out the selections and revert the SuperSelect to its deselected state.
+
+![Flow showing the behavior of the clear selected button and how it reverts the SuperSelect's state to the default one](/assets/components/form/super-select/super-select-flow-clear-selected.png)
+
+##### Footer with custom content
+
+In either of the two variants, you are able to fill the footer with custom content. This is helpful when wanting to mimic the footer but still customize its contents.
+
+![Showing the footer with custom content inside.](/assets/components/form/super-select/super-select-single-multiple-footer-custom-content.png =836x*) 
+
+#### CustomContent
+
+If enabled, custom content will occupy this space.
+
+![Showing the afterOptions slot with custom content](/assets/components/form/super-select/super-select-after-options-custom-content.png =836x*)
+
+## List
+
+### Positioning
+
+Lists can be positioned to the left, center or right of the trigger, and above or below the trigger to fit more appropriately within the UI. These options are only available when `matchTriggerWidth` is set to `false`.
+
+![Showing the different list positions (left, center and right + top and bottom)](/assets/components/form/super-select/super-select-list-positioning.png =836x*)
+
+### Width
+
+#### Match the trigger
+
+By default, the width of the list matches the width of the trigger.
+
+![Showing the list's width matching the trigger's](/assets/components/form/super-select/super-select-list-width-matching.png)
+
+#### Auto width
+
+If the list width is set to auto, it will expand to be the size of the longest list item.
+
+![Showing the list's auto width, resizing to the longest list item](/assets/components/form/super-select/super-select-list-width-auto.png)
+
+## ListItem Types
+
+There are four types of ListItems, two of which are variant specific.
+
+- `Checkbox` is used to clarify to the user that multiple selections are permitted. This is only allowed within the multiple variant.
+- `Checkbox` is used to delineate to the user that multiple selections within the list is permitted. This is only allowed within the multiple variant.
+- `Title` is used to group ListItems together and help visually organize groups.
+- `Separator` is used to create a visual separation between groups.
+
```

Comment:
how are the two checkbox types different?

Reply:
typo

Should be:
checkmark = Single
checkbox = Multiple

Reply:
Oops, ill fix this, this was accidental when I was updating things. Thanks for the catch.